tests/integration/kubernetes/k8s-sealed-secret.bats

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bats
+# Copyright 2024 IBM Corporation
+# Copyright 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Test for Sealed Secret feature of CoCo
+#
+
+load "${BATS_TEST_DIRNAME}/lib.sh"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/confidential_kbs.sh"
+
+export KBS="${KBS:-false}"
+export KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
+export AA_KBC="${AA_KBC:-cc_kbc}"
+
+setup() {
+	[ "${KATA_HYPERVISOR}" = "qemu-coco-dev" ] || skip "Test not ready yet for ${KATA_HYPERVISOR}"
```

Comment:
Could this work on TDX/SE (who have attestation support working) now? If so we could swap this to 
`is_confidential_runtime_class || skip "Test not supported for ${KATA_HYPERVISOR}."` like the attestation-agent and let the KBS check rule out platforms that don't support that.
If you have already checked and that they are failing currently, that I understand getting it working for just the non-TEE first, so this is fine.

Reply:
I was thinking we would enable the TEEs separately. in theory we should run this test on all the confidential runtime classes as long as KBS is true (because it depends on attestation), but seeing the current state of the CI I'm not sure we're ready yet.